From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Shelley Vohr <shelley.vohr@gmail.com>
Date: Tue, 2 Oct 2018 11:39:58 -0700
Subject: feat: add new built_with_electron variable to config.gypi

This allows 3rd-party native modules to know whether they're being built
against Electron.

diff --git a/tools/generate_config_gypi.py b/tools/generate_config_gypi.py
index 01f62d4ae6e3b9d539444e3dff069f0011353caa..d8b279f590c115108d5dca879747de7b0c9f1934 100644
--- a/tools/generate_config_gypi.py
+++ b/tools/generate_config_gypi.py
@@ -5,7 +5,7 @@ import sys
 def main(args):
   out = args[0]
   with open(out, 'w') as f:
-    f.write("{'variables':{}}\n")
+    f.write("{'variables':{'built_with_electron': 1}}\n")
 
 if __name__ == '__main__':
   main(sys.argv[1:])
